package.json

```diff
@@ -57,5 +58,76 @@
     "src/**/*.(js|jsx)": [
       "eslint src/**/*.{js|jsx}"
```

Comment:
IMO it is worth to have stylelinting triggered on staged files also. What's more we should have new (`stylelint`?) script defined in package.json.

Reply:
Because thanks to that CI will could trigger it also.